src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/takecard/TakeCardStyles.kt

```diff
@@ -1,47 +1,90 @@
 package org.wycliffeassociates.otter.jvm.app.widgets.takecard
 
-import de.jensd.fx.glyphs.materialdesignicons.MaterialDesignIcon
-import de.jensd.fx.glyphs.materialdesignicons.MaterialDesignIconView
-import de.jensd.fx.glyphs.materialicons.MaterialIcon
-import de.jensd.fx.glyphs.materialicons.MaterialIconView
 import javafx.scene.effect.DropShadow
 import javafx.scene.paint.Color
 import javafx.scene.text.FontPosture
 import javafx.scene.text.FontWeight
 import org.wycliffeassociates.otter.jvm.app.images.ImageLoader
+import org.wycliffeassociates.otter.jvm.app.theme.AppTheme
 import tornadofx.*
 
 class TakeCardStyles : Stylesheet() {
+
     companion object {
-        val defaultTakeCard by cssclass()
+        val scriptureTakeCard by cssclass()
+        val resourceTakeCard by cssclass()
         val badge by cssclass()
         val iconStyle by cssclass()
         val content by cssclass()
         val takeNumberLabel by cssclass()
         val timestampLabel by cssclass()
         val defaultButton by cssclass()
         val takeprogressBar by cssclass()
+        val editButton by cssclass()
+        val topHalf by cssclass()
         val defaultGreen : Color = c("#58BD2F")
+        val grey = c("#C9C8C8")
 
-        fun pauseIcon() = MaterialIconView(MaterialIcon.PAUSE, "30px")
-        fun deleteIcon() = MaterialIconView(MaterialIcon.DELETE, "25px")
-        fun badgeIcon() = MaterialDesignIconView(MaterialDesignIcon.CREATION, "18px")
-        fun playIcon() = MaterialIconView(MaterialIcon.PLAY_ARROW, "25px")
-        fun editIcon() = MaterialIconView(MaterialIcon.EDIT, "25px").apply { fill = defaultGreen }
         fun draggingIcon() = ImageLoader.load(
-                ClassLoader.getSystemResourceAsStream("images/baseline-drag_indicator-24px.svg"),
-                ImageLoader.Format.SVG
+            ClassLoader.getSystemResourceAsStream("images/baseline-drag_indicator-24px.svg"),
+            ImageLoader.Format.SVG
         )
     }
 
     init {
-        defaultTakeCard {
+        takeNumberLabel {
+            fontSize = 16.px
+            fontWeight = FontWeight.BOLD
+        }
+
+        takeprogressBar {
```

Comment:
if we're camelcasing this, replace everywhere with takeProgressBar

Reply:
done